### PR TITLE
Converted userRole: admin to an admin scope.

### PR DESCRIPTION
**Pull request by barankyle**
## Summary

Admin status was defined by user.userRole being 'admin'. This has been changed so that
admin status is a scope 'admin:admin'.

Made a script to convert all existing `userRole: admin`s to `userRole: user` and add an
admin scope for them.

## References

closes #_insert number here_

## Checklist
- [x] If this PR is still a WIP, convert to a draft
- [x] [ensure all checks pass](https://github.com/XRFoundation/XREngine/wiki/Testing-&-Contributing)
- [x] When this PR is ready, mark it as "Ready for review"
- [x] Changes have been manually QA'd
- [ ] Changes reviewed by at least 2 approved reviewer

## QA Steps

_List any additional steps required to QA the changes of this PR, as well as any supplemental images or videos._

### CODE REVIEW

**Comment by barankyle** [comment]
Realized right after I opened this that we wanted to convert userRole to isGuest, doing that now.

**Review by dmitrizagidulin — APPROVED** [review]
:+1:

File: packages/client-core/src/admin/components/Users/UserDrawer.tsx [hunk 1]
```diff
@@ -234,15 +211,15 @@ const UserDrawer = ({ open, mode, selectedUser, onClose }: Props) => {
...
-          label={t('admin:components.user.userRole')}
-          value={state.userRole}
-          error={state.formErrors.userRole}
-          menu={userRoleMenu}
-          disabled={viewMode}
-          onChange={handleChange}
-        />
+        {viewMode && (
+          <>
+            <InputText
```
Comment (HexaField) [inline packages/client-core/src/admin/components/Users/UserDrawer.tsx:216]: this should be a boolean checkbox input if possible

**Comment by HexaField** [comment]
Screenshot from 2022-08-05 09-45-55
profile modal does not reflect user role. i ran `npm run make-user-admin ...` on this user id, it was successful and i can access the admin panel, but the `Guest` text has not update.

**Review by HexaField — APPROVED** [review]

**Pull request merged**